data/influx/influx.go

```diff
@@ -38,16 +38,24 @@ func (s *Influx) query(query string, database string) client.Query {
 }
 
 // Query executes the provided query string and returns the results as a JSON object
```

Comment:
Since we're not returning JSON for a REST API, do we need to marshal to JSON? It seems that you might want to use helpers that transform client.Response to typed objects. Or perhaps an expedient method for now is to leverage json and then unmarshal into protocol buffer messages used by the rpc service using [jsonpb](https://godoc.org/github.com/golang/protobuf/jsonpb). This would make it a bit more like the data/storage/etcd package that accepts an out parameter that is an instance of the actual empty proto.Message instance to unmarshal the result into.